apps/studio/components/interfaces/Auth/Policies/PolicyTableRow/PolicyTableRowHeader.tsx

```diff
@@ -70,21 +76,41 @@ const PolicyTableRowHeader = ({
                 {table.rls_enabled ? 'Disable RLS' : 'Enable RLS'}
               </ButtonTooltip>
             )}
-            {!isAiAssistantEnabled && (
-              <ButtonTooltip
-                type="default"
-                disabled={!canToggleRLS}
-                onClick={() => onSelectCreatePolicy()}
-                tooltip={{
-                  content: {
-                    side: 'bottom',
-                    text: 'You need additional permissions to create RLS policies',
-                  },
-                }}
-              >
-                Create policy
-              </ButtonTooltip>
-            )}
+            <ButtonTooltip
+              type="default"
+              disabled={!canToggleRLS}
+              onClick={() => onSelectCreatePolicy()}
+              tooltip={{
+                content: {
+                  side: 'bottom',
+                  text: !canToggleRLS
+                    ? 'You need additional permissions to create RLS policies'
+                    : undefined,
+                },
+              }}
+            >
+              Create policy
+            </ButtonTooltip>
+
+            <Tooltip_Shadcn_>
+              <TooltipTrigger_Shadcn_ asChild>
+                <Button
+                  type="default"
+                  className="px-1"
+                  onClick={() => {
+                    onSelectCreatePolicy()
+                    setEditView('conversation')
+                  }}
+                >
+                  <AiIconAnimation className="scale-75 [&>div>div]:border-black dark:[&>div>div]:border-white" />
+                </Button>
+              </TooltipTrigger_Shadcn_>
+              <TooltipContent_Shadcn_ side="top">
+                {!canToggleRLS
+                  ? 'You need additional permissions to create RLS policies'
+                  : 'Create with the Supabase Assistant'}
```

Comment:
"Create (or generate?) with Supabase Assistant" sounds more natural to me here but i may be wrong

Reply:
yup sounds good